src/apps/aave/shortcuts.ts

```diff
@@ -216,6 +219,72 @@ const hook: ShortcutsHook = {
           ]
         },
       }),
+      createShortcut({
+        id: 'swap-deposit',
```

Comment:
open to suggestions for a better id / category for this

Reply:
`zap-in/out` works, should we also change the input param from `swapFromToken` to `zapFromToken`?

Reply:
I like `swap-deposit`. Its concise and obvious from the name what its doing.

Reply:
brought this up in activation sync and people seemed to prefer `swap-deposit`, so keep it as is.

Reply:
Ok that’s fair. One is directly understandable for people not familiar with crypto.
Whereas, zap is used in several crypto products like Beefy, [Yearn](https://andrecronje.gitbook.io/yearn-finance/products/zap), Zapper, [PancakeSwap](https://docs.pancakeswap.finance/products/pancakeswap-exchange/zap/zap-v2), [KyberSwap](https://docs.kyberswap.com/getting-started/foundational-topics/decentralized-finance/zap), [Krystal](https://x.com/krystaldefi/status/1780162409333657954) or [Symbiosis Finance](https://app.symbiosis.finance/zap). And may become the industry term for this.